cfme/utils/appliance/__init__.py

```diff
@@ -1012,17 +1012,25 @@ def ssh_client(self):
             }
         if self.is_dev:
             connect_kwargs.update({'is_dev': True})
-        ssh_client = ssh.SSHClient(**connect_kwargs)
-        try:
-            ssh_client.get_transport().is_active()
-            logger.info('default appliance ssh credentials are valid')
-        except Exception as e:
-            if self.is_dev:
-                raise Exception('SSH access on a dev alliance, (unsupported)')
-            logger.error(e)
-            logger.error('default appliance ssh credentials failed, trying establish ssh connection'
-                         ' using ssh private key')
-            ssh_client = self.ssh_client_with_privatekey()
+
+        def create_ssh_connection():
+            ssh_client = ssh.SSHClient(**connect_kwargs)
+            try:
+                ssh_client.get_transport().is_active()
+                logger.info('default appliance ssh credentials are valid')
+            except Exception as e:
+                if self.is_dev:
+                    raise Exception('SSH access on a dev alliance, (unsupported)')
```

Comment:
This is a typo in the original message anyway - should be `'SSH access unsupported on dev appliance'`

Reply:
It also makes more sense to check this before even calling wait_for on this method, since we don't expect it to work in the first place?

Reply:
@mshriver I don't know. I guess the check is in the `except` for some reason. Perhaps we want to shoot the warning only when things go wrong.